.github/workflows/frontend-applications/patch-generic.yml

```diff
@@ -0,0 +1,132 @@
+name: Patch
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: 'Release version (format: vX.X)'
+        required: true
+        type: string
+      dockerImage:
+        description: 'Name of the Docker image (e.g., organization/app-name)'
+        required: true
+        type: string
+      dockerUsername:
+        required: true
+        type: string
+      githubappId:
+        description: 'GitHub App ID for authentication'
+        required: true
+        type: string
+    secrets:
+      githubappPrivateKey:
+        required: true
+      docker-token:
+        required: true
+      sonar-token:
+        required: true
+
+env:
+  NODE_OPTIONS: '--max_old_space_size=4096'
+
+jobs:
+  patch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate GitHub App Token
+        id: app-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ inputs.githubappId }}
+          private-key: ${{ secrets.githubappPrivateKey }}
+
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ steps.app-token.outputs.token }}
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Parse release version
+        run: |
+          regex="^v([0-9]+)\.([0-9]+)$"
+          if ! [[ ${{ inputs.releaseVersion }} =~ $regex ]]; then
+            echo "ERROR: Release version must match the format vX.X (e.g., v2.8)"
+            echo "Received: ${{ inputs.releaseVersion }}"
+            exit 1
+          fi
+          
+          MAJOR="${BASH_REMATCH[1]}"
+          MINOR="${BASH_REMATCH[2]}"
+          
+          echo "RELEASE_BRANCH=release-v${MAJOR}.${MINOR}" >> $GITHUB_ENV
+
+      - name: Verify Release Branch Exists
+        run: |
+          if ! git ls-remote --quiet --exit-code origin refs/heads/${{ env.RELEASE_BRANCH }} >/dev/null 2>&1; then
+            echo "ERROR: Release branch ${{ env.RELEASE_BRANCH }} does not exist"
+            echo "Please ensure you have created the release branch before running a patch"
+            exit 1
+          fi
+          
+          echo "Release branch ${{ env.RELEASE_BRANCH }} found - continuing with patch"
+
+      - name: Checkout on existing release branch
+        run: |
+          git checkout -b ${{ env.RELEASE_BRANCH }}
```

Comment:
Can you just copy/paste this from backend-applications ? It's hard to see if it does the same if it is slightly different. And for the checkout it shouldn't be -b because it is supposed to exist already.